server/notifications_slack.go

```diff
@@ -189,6 +189,14 @@ func createSlackIntegration(keys map[string]string, properties map[string]bool,
 		if err != nil {
 			return slackIntegration, err
 		}
+
+		if value, ok := keys["auth_token"]; ok {
+			encryptedValue, err := EncryptAES([]byte(value))
+			if err != nil {
+				return models.Integration{}, err
+			}
+			keys["auth_token"] = encryptedValue
```

Comment:
this is not good, you are set it here then keys go into the integrationToUpdate var and then this is broadcast to the clients with an encrypted value